core-bundle/src/Resources/contao/languages/en/default.xlf

```diff
@@ -1950,7 +1950,13 @@
         <source>There is at least one regular user with access to the template editor. Since templates are PHP files, these users implicitly have full control over the system!</source>
       </trans-unit>
       <trans-unit id="MSC.groupTemplateEditor">
-        <source>There is at least one user group granting access to the template editor. Since templates are PHP files, users of these groups implicitly have full control over the system!</source>
+        <source>There is at least one user group with access to the template editor. Since templates are PHP files, users of these groups implicitly have full control over the system!</source>
```

Comment:
I liked the previous text better (*granting* for groups)

Reply:
Changed in 1fa2bb6439a01aff9320f406e950ea5f70efd249.